src_tests/test_simulation_strategies.py

```diff
@@ -52,17 +53,19 @@ def setUp(self) -> None:
         self._probability_of_error_in_distribution_calculation = 1e-3
 
         self._distance_calculation_initial_state = [1, 1, 1, 1, 0]
+        #self._distance_calculation_initial_state = [1, 1, 0, 0, 0]
```

Comment:
A fragment of code that is commented out. Is that intended? How about removing it?

Reply:
Removed.